CHANGELOG.md

```diff
@@ -2552,3 +2553,4 @@
 [@kevindew]: https://github.com/kevindew
 [@lucasuyezu]: https://github.com/lucasuyezu
 [@breckenedge]: https://github.com/breckenedge
+[@enriikke] https://github.com/enriikke
```

Comment:
Thank you!

Really awesome test coverage. I modified the CHANGELOG last, after I had already ran the tests and didn't even occurred to me to run them again. 👍